README.md

```diff
@@ -10,46 +10,68 @@ This library provides i18n functionality for Python 3 out of the box. The usage
 
 Just run
 
-    pip install i18nice
+```shell
+pip install i18nice
+```
 
 If you want to use YAML to store your translations, use
 
-    pip install i18nice[YAML]
+```shell
+pip install i18nice[YAML]
+```
 
 ## Usage
 ### Basic usage
 
 The simplest, though not very useful usage would be
 
-    import i18n
-    i18n.add_translation('foo', 'bar')
-    i18n.t('foo') # bar
-
+```python
+import i18n
+i18n.add_translation('foo', 'bar')
+i18n.t('foo') # bar
+```
+    
 ### Using translation files
 
 YAML and JSON formats are supported to store translations. With the default configuration, if you have the following `foo.en.yml` file
 
-    en:
-      hi: Hello world !
+```yaml
+en:
+  hi: Hello world !
+```
 
-in `/path/to/translations` folder, you simply need to add the folder to the translations path.
+Or a JSON file `foo.en.json`
+
+```json
+{
+  "en": {
+    "hi": "Hello world !"
+  }
+}
+```
 
-    import i18n
-    i18n.load_path.append('/path/to/translations')
-    i18n.t('foo.hi') # Hello world !
+in `/path/to/translations` folder, you simply need to add the folder to the translations path.
 
+```python
+import i18n
+i18n.load_path.append('/path/to/translations')
+i18n.t('foo.hi') # Hello world !
+```
+        
 Please note that YAML format is used as default file format if you have `yaml` module installed.
 If both `yaml` and `json` modules available and you want to use JSON to store translations, explicitly specify that: `i18n.set('file_format', 'json')`
 
 **!WARNING!**
-`yaml.FullLoader` is no longer used by default.
+`yaml.FullLoader` is no longer used by default due to safety reason.
```

Comment:
This is not quite true. Usually translation files are a part of project itself, there is no reason to not trust them. It was changed because of value parsing (#4)

Reply:
Sorry for mistake, this was fixed.